db/migrate/20180131112642_change_dimensions_items_fields.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeDimensionsItemsFields < ActiveRecord::Migration[5.1]
```

Comment:
This migration suggests it is modifying the `dimensions_items` table however it changes `dimensions_items_temps` and is identical to `ChangeDimensionsItemsTempsFields`. It looks like some copy and pasting has gone wrong!